app/app.py

```diff
@@ -45,7 +45,7 @@ def __init__(self, lists):
             engine = wizard.CONNECTION.engine
             options = wizard.CONNECTION.option
             opts = dict()
-            for key in options.keys():
+            for key in list(options.keys()):
```

Comment:
You could also replace the whole loop with a dictionary comprehension if you don't need to support Python 2.6:
```python
engine.opts = {key: value.GetValue() for key, value in options.items()}
```